protocol/communities/community.go

```diff
@@ -2154,12 +2191,20 @@ func (o *Community) ValidateEvent(event *CommunityEvent, signer *ecdsa.PublicKey
 		return err
 	}
 
-	member := o.getMember(signer)
-	if member == nil {
+	eventSender := o.getMember(signer)
+
+	pk, err := common.HexToPubkey(event.MemberToAction)
+	if err != nil {
+		return err
+	}
+
+	member := o.getMember(pk)
```

Comment:
```suggestion
	eventTarget := o.getMember(pk)
```
Could increase readability a bit IMO.

Reply:
fixed